mysql-test/suite/rpl/t/rpl_delayed_parallel_slave_sbm-slave.opt

```diff
@@ -0,0 +1 @@
+--slave-parallel-threads=4
```

Comment:
As slave is getting restarted with a new `$master_delay`, let's not create a new file and set the var in the test.